MIGRATION_GUIDE.md

```diff
@@ -16,6 +16,14 @@
 - `.blr-sm` removed
 - `.blr-lg` removed
 
+### Components
+
+#### Popover
+- `s-popover--arrow` removed
```

Comment:
Rather than limit ourselves to write what changed I would recommend to add here what the consumers are expected to do. After all this is not a changelog but a migration guide doc. 🙂

So in this case it could be something like. 
```
The new popovers no longer include an arrow element. The `s-popover--arrow` css class has been removed, and any markup using it (e.g. `<div class="s-popover--arrow"></div>`) should be deleted from the codebases as part of the migration.
```